src/Maestro/Client/src/Generated/Subscriptions.cs

```diff
@@ -123,7 +125,11 @@ public Subscriptions(MaestroApi client)
             {
                 _url.AppendQuery("enabled", Client.Serialize(enabled));
             }
-            _url.AppendQuery("api-version", MaestroApi.Serialize(apiVersion));
+            if (sourceEnabled != default)
```

Comment:
A nit: when reading this, I have to go back to the variable definition to see if default means nullable or false. Structuring the condition as `is null` would more explicitly the intent more clearly, but it's more of a GH review issue as it would be clear when reading the code in VS.

Reply:
This is also auto-generated code

Reply:
We could change `.editorconfig` to change the `default(T)` preference